### ISSUE TITLE
Autocomplete method in Anaconda for Sublime text stuck on 'OS' module

### ISSUE THREAD

**Issue by GlassGruber** [pdbpp/pdbpp#68]
Hey there, 
first of all thank you for this, it's really a great tool for novice like me ;)

I'm finding myself quite at ods with the implementation of the autocomplete feature of Anaconda plugin on Sublime Text 3. 
For what I can tell, Anaconda can auto complete `pdb.` (note the dot) only with `os` module, I guess because the file it is reading actually is in this folder `\site-packages\_pdbpp_path_hack` with the following content:
```
# this file is needed to hijack pdb without eggs
import os.path
exec(compile(open(os.path.join(
    os.path.dirname(os.path.dirname(__file__)), 'pdb.py')).read(), os.path.join(
    os.path.dirname(os.path.dirname(__file__)), 'pdb.py'), 'exec'))
```

So basically if I try to auto complete `pdb.` expecting a list with for instance `set_trace`, I get in this list only the  `os` module along with builtin class functions like `__doc__`, `__file__`, etc...

Even though the autocomplete issue nuissance, I've installed the module sucessfully with _pip_ and it's working correctly.

I'm sure I'm overlooking something, or maybe this is "by design" since basically you usually call the `set_trace` method and that's it. Hopefully nothing so obvious that a more accurate RTFM would have fixed :)

Thank you!

**Comment by antocuni** [comment]
This looks like a bug/limitation of your autocomplete plugin to me. It is also the reason why I don't like autocompleters which rely on static analysis: they can work only on a limited number of cases, but they have no chance to work in the general case, in a language which is as dynamic as Python.
